### PR TITLE
Sync 6 files under configure-pod-container/ with 1.22 version

### PR DESCRIPTION
**Pull request by superleo**
for issue: https://github.com/kubernetes/website/issues/29329        

modified:   content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md
        modified:   content/zh/docs/tasks/configure-pod-container/configure-runasusername.md
        modified:   content/zh/docs/tasks/configure-pod-container/configure-service-account.md
        modified:   content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md
        modified:   content/zh/docs/tasks/configure-pod-container/quality-service-pod.md
        modified:   content/zh/docs/tasks/configure-pod-container/security-context.md

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -672,35 +672,51 @@ to resolve it.
...
+In 1.21 and beyond, when the feature gate `ProbeTerminationGracePeriod` is
+enabled, users can specify a probe-level `terminationGracePeriodSeconds` as
+part of the probe specification. When the feature gate is enabled, and both a
+pod- and probe-level `terminationGracePeriodSeconds` are set, the kubelet will
+use the probe-level value.
 -->
 在 1.21 版之前，pod 级别的 `terminationGracePeriodSeconds` 被用来终止
 未能成功处理活跃性探测或启动探测的容器。
 这种耦合是意料之外的，可能会导致在设置了 pod 级别的 `terminationGracePeriodSeconds` 后，
 需要很长的时间来重新启动失败的容器。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:687]: 以上删除

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:687]: 678-682 重新翻译

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -672,35 +672,51 @@ to resolve it.
...
+If you have existing Pods where the `terminationGracePeriodSeconds` field is set and
+you no longer wish to use per-probe termination grace periods, you must delete
+those existing Pods.
 -->
-在1.21中，启用特性标志 `ProbeTerminationGracePeriod` 后，
-用户可以指定一个探测器级别的 `terminationGracePeriodSeconds` 作为探测器规格的一部分。
-当该特性标志被启用时，若同时设置了 Pod 级别和探测器级别的 `terminationGracePeriodSeconds`，
-kubelet 将使用探测器级的值。
+如果你已经为现有 Pod 设置了 “terminationGracePeriodSeconds” 字段并且
+不再希望使用针对每个探针终止宽限期，则必须删除那些现有的 Pod。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:705]:
```suggestion
不再希望使用针对每个探针的终止宽限期，则必须删除那些现有的 Pod。
```

File: content/zh/docs/tasks/configure-pod-container/configure-service-account.md [hunk 3]
```diff
@@ -540,15 +540,18 @@ JSON Web Key Set（JWKS）。
...
 `system:authenticated` or `system:unauthenticated` depending on their security
 requirements and which external systems they intend to federate with.
 -->
-集群包括一个默认的 RBAC ClusterRole，
-名为 `system:service-account-issuer-discovery`。
-默认情况下不提供角色绑定对象。
-举例而言，管理员可以根据其安全性需要以及期望集成的外部系统选择是否将该角色绑定到
+集群包括一个的默认 RBAC ClusterRole, 名为 `system:service-account-issuer-discovery`。 
+默认的 RBAC ClusterRoleBinding 将此角色分配给 `system:serviceaccounts` 组，
+所有服务帐户隐式属于该组。 这允许在集群上运行的 Pod 访问通过他们挂载的服务帐户令牌的服务帐户发现文档。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-service-account.md:553]:
```suggestion
所有服务帐户隐式属于该组。这使得集群上运行的 Pod 能够通过它们所挂载的服务帐户令牌访问服务帐户发现文档。
```

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 4]
```diff
@@ -299,6 +299,42 @@ and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
...
+`fsGroup`.
+
+Please refer to the [KEP](https://github.com/gnufied/enhancements/blob/master/keps/sig-storage/2317-fsgroup-on-mount/README.md)
+and the description of the `VolumeCapability.MountVolume.volume_mount_group`
+field in the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume)
+for more information.
+-->
+
+如果你部署了一个[容器存储接口 (CSI)](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+驱动支持 `VOLUME_MOUNT_GROUP` `NodeServiceCapability` ，
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:327]:
```suggestion
驱动支持 `VOLUME_MOUNT_GROUP` `NodeServiceCapability`，
```

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 5]
```diff
@@ -299,6 +299,42 @@ and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
...
+在`securityContext` 中指定 `fsGroup` 来设置文件所有权和权限的过程将由 CSI 驱动
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:328]:
```suggestion
在 `securityContext` 中指定 `fsGroup` 来设置文件所有权和权限的过程将由 CSI 驱动
```

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 6]
```diff
@@ -299,6 +299,42 @@ and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
...
+而不是 Kubernetes 来执行，前提是 Kubernetes 的 `DelegateFSGroupToCSIDriver` 
+特性门控已启用。在这种情况下，由于 Kubernetes 不执行任何
+所有权和权限更改，`fsGroupChangePolicy` 不会生效，并且
+按照 CSI 的规定，CSI 驱动应该使用 `fsGroup`  来挂载卷，从而产生了一个被 `fsGroup` 可读/可写的卷。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:332]:
```suggestion
按照 CSI 的规定，CSI 驱动应该使用所指定的 `fsGroup` 来挂载卷，从而生成了一个对 `fsGroup` 可读/可写的卷。
```

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 7]
```diff
@@ -299,6 +299,42 @@ and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
...
+
+更多的信息请参考 [KEP](https://github.com/gnufied/enhancements/blob/master/keps/sig-storage/2317-fsgroup-on-mount/README.md)
+和[CSI 规范](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume) 中的字
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:335]:
```suggestion
和 [CSI 规范](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume) 中的字
```

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 2 above)
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:705]: fixed

File: content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 1 above)
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:687]: fixed

File: content/zh/docs/tasks/configure-pod-container/configure-service-account.md (hunk 3 above)
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-service-account.md:553]: fixed

**Review by chenrui333 — APPROVED** [review]
/lgtm
/approve

---

Thanks!

**Pull request merged**